help/include/topic-notifications-intro.md

```diff
@@ -0,0 +1,23 @@
+To avoid receiving notifications messages you are not interested in, Zulip lets
+you mute topics and streams. Muting a stream automatically mutes all topics in
```

Comment:
Maybe "Muting a stream effectively mutes all topics ..."? The current phrasing could be interpreted as it changing the setting for those topics, which isn't what it does.